terraform/github/data-catalogue.tf

```diff
@@ -33,6 +33,23 @@ module "data_catalogue_repository" {
   }
 }
 
+module "datahub_custom_api_source" {
+  source = "./modules/repository"
+
+  name        = "data-catalogue"
+  description = "Custom ingestion source for Datahub to consume from https://data.justice.gov.uk/api"
+  topics      = ["data-catalogue"]
+
+  use_template        = true
+  template_repository = "data-platform-app-template"
```

Comment:
instead of using the app template, you can drop this off and it will use the MoJ default template

```suggestion
```